examples/vue/src/pages/ui/components/authenticator/useAuthenticator/home/index.vue

```diff
@@ -0,0 +1,26 @@
+<script setup lang="ts">
+import { toRefs } from 'vue';
+import { useRouter } from 'vue-router';
+
+import { Amplify } from 'aws-amplify';
+import { Authenticator, useAuthenticator } from '@aws-amplify/ui-vue';
+import '@aws-amplify/ui-vue/styles.css';
+import aws_exports from '../aws-exports';
+
+Amplify.configure(aws_exports);
+
+const router = useRouter();
+const { user, signOut } = toRefs(useAuthenticator());
+const handleClick = (event: Event) => {
+  event.preventDefault();
+  signOut.value();
+  router.push('../useAuthenticator');
+};
+</script>
+
+<template>
+  <!-- TODO: this authenticator shouldn't be needed -->
+  <authenticator></authenticator>
```

Comment:
Vue still needed this hidden `authenticator`, because authenticator was needed to interpret and start the machine in the first place. This seemed like a much bigger refactor, so left this as-is.